utils/synth_wav.sh

```diff
@@ -0,0 +1,250 @@
+#!/bin/bash
+
+# Copyright 2019 Nagoya University (Takenori Yoshimura)
+#  Apache 2.0  (http://www.apache.org/licenses/LICENSE-2.0)
+
+if [ ! -f path.sh ] || [ ! -f cmd.sh ]; then
+    echo "Please change directory to e.g., egs/ljspeech/tts1"
+    exit 1
+fi
+
+. ./path.sh
+
+# general configuration
+backend=pytorch
+stage=0        # start from 0 if you need to start from data preparation
+stop_stage=100
+ngpu=0         # number of gpus ("0" uses cpu, otherwise use gpu)
+debugmode=1
+verbose=1      # verbose option
+
+# feature configuration
+fs=24000      # sampling frequency
+fmax=""       # maximum frequency
+fmin=""       # minimum frequency
+n_mels=80     # number of mel basis
+n_fft=1024    # number of fft points
+n_shift=256   # number of shift points
+win_length="" # window length
+cmvn=
+
+# dictionary related
+dict=
+
+# x-vector related
+use_input_wav=false
+input_wav="input.wav"
+
+# decoding related
+synth_model=
+decode_config=
+decode_dir=decode
+griffin_lim_iters=1000
+
+# download related
+models=libritts.v1
+
+. utils/parse_options.sh || exit 1;
+
+# make shellcheck happy
+train_cmd=
+decode_cmd=
+
+. ./cmd.sh
+
+txt=$1
+download_dir=${decode_dir}/download
+
+if [ $# -gt 1 ]; then
+    echo "Usage: $0 <wav>"
+    exit 1;
+fi
+
+set -e
+set -u
+set -o pipefail
+
+case "${models}" in
+    "libritts.v1")
```

Comment:
Even when I want to use my own model, `use_input_wav` is always set as `true`.

Reply:
To solve the problem, I set the initial value of `models` to empty.

Reply:
Fixed